packages/eth-provider-types/index.ts

```diff
@@ -284,9 +284,9 @@ export interface UserOperation {
   callData: string;
   callGasLimit?: string;
   verificationGasLimit?: string;
-  preVerificationGasLimit?: string;
```

Comment:
this is versioned so it won't break clients right?

Reply:
No, it shouldn't - but also, we're the only ones using this right now, and the package is currently in `prerelease` mode.